app/src/main/java/ch/epfl/skysync/util/FormsUtil.kt

```diff
@@ -0,0 +1,41 @@
+package ch.epfl.skysync.util
+
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+import java.util.Date
+
+/** Useful methods for input validation */
+fun inputValidation(vararg errors: Boolean): Boolean {
```

Comment:
Name could be misleading, would rename it `hasAnyError`

Reply:
Done